webpack/assets/javascripts/react_app/components/common/forms/Actions.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Button } from 'patternfly-react';
+
```

Comment:
nitpick: redundant empty line?

Reply:
I always split the `3rd-party` modules from the `local-modules` by empty line :smiley: